components/ingest-service/backend/elastic/elastic.go

```diff
@@ -27,6 +27,33 @@ type Backend struct {
 	client      *elastic.Client
 }
 
+// This method will support adding a document with a specified id
+func (es *Backend) upsertComplianceRunInfo(ctx context.Context, mapping mappings.Mapping, id string, runDateTime time.Time) error {
```

Comment:
This function name is not correct. It needs to be
```suggestion
func (es *Backend) upsertNodeRunInfo(ctx context.Context, mapping mappings.Mapping, id string, runDateTime time.Time) error {
```

Reply:
fixed

thank you!